src/main/java/org/openlmis/stockmanagement/service/StockCardSummariesService.java

```diff
@@ -144,8 +145,9 @@ public StockCardSummaries findStockCards(StockCardSummariesV2SearchParams params
 
     stockCards.stream().forEach(StockCard::calculateStockOnHand);
 
-    StockCardSummaries result = new StockCardSummaries(approvedProducts.getContent(), stockCards,
-        orderableFulfillMap, params.getAsOfDate(), approvedProducts.getTotalElements());
+    StockCardSummaries result = new StockCardSummaries(
```

Comment:
we call the **approvedProducts.getOrderablesPage()** twice so from what I see the orderable page object will be created also twice

Reply:
You're probably right. I'll extract it above so it'll be used once.